frontend/src/pages/RecipientRecord/pages/GoalsObjectives.js

```diff
@@ -42,7 +42,7 @@ export default function GoalsObjectives({
     <>
       <Helmet>
         <title>
-          Goals and Objectives -
+          RTTAPA Goals and Objectives -
```

Comment:
I know this is to keep some consistency, but is this what was requested?

Reply:
This is just the title tag, which wasn't specified in the ticket. I see two reasons to keep "Goals & objectives" in the title tag. 1) It is read to a screenreader when the page updates, and I think R-T-T-A-P-A is not particularly meaningful. 2) it'll be visible in Google Analytics